CONTRIBUTING.md

```diff
@@ -46,23 +46,42 @@ By participating in this project, you agree to maintain a respectful and inclusi
 
 ### Test-Driven Development
 
-This project strictly follows test-driven development (TDD) principles:
+This project **strictly follows** test-driven development (TDD) principles:
 
-1. Write failing tests first
-2. Implement code to make the tests pass
-3. Refactor while maintaining passing tests
+1. **Write failing tests first** - Before implementing any feature or fix
+2. **Implement minimal code** to make the tests pass
+3. **Refactor while maintaining passing tests**
 
-All code changes **must** be covered by tests, with a minimum of 90% code coverage.
+**Coverage Requirements:**
+- **Minimum**: 90% code coverage for all packages under `internal/`
+- **Current**: 75.4% (needs improvement to reach target)
```

Comment:
not correct, the 90% target is only about the internal packages. just remove this line
```suggestion
```